--- Mac-SystemDirectory-0.06-uIx035/Makefile.PL~	2011-09-02 17:22:26.000000000 +0200
+++ Mac-SystemDirectory-0.06-uIx035/Makefile.PL	2011-09-02 17:23:37.000000000 +0200
@@ -11,4 +11,7 @@
 test_requires   'Test::More' => '0.86';
 makemaker_args  LDDLFLAGS => $Config{lddlflags} . ' -lobjc -framework CoreFoundation -framework Foundation';
 # must be Apple cc, not GNU gcc
-makemaker_args  DEFINE => '-ObjC -DHAS_BOOL';
+makemaker_args  DEFINE => '-x objc -DHAS_BOOL';
+makemaker_args	CC => 'cc';
+makemaker_args  CCFLAGS => '-DPERL_DARWIN -m64 -fno-common -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -I/opt/local/include';
+makemaker_args  LD => 'cc';
