2005-09-10 20:47:58 rurban@cpan.org

	* Windows: fallback on ncpu failure

--- Test-Smoke-1.19/lib/Test/Smoke/SysInfo.pm.orig	2004-07-24 16:26:25.000000000 +0000
+++ Test-Smoke-1.19/lib/Test/Smoke/SysInfo.pm	2005-09-10 18:46:28.413250000 +0000
@@ -528,6 +528,7 @@
             hex $Win32::TieRegistry::Registry->{ $mhzkey };
         $cpu = $cpustr;
         $ncpu = keys %{ $Win32::TieRegistry::Registry->{ $basekey } };
+	$ncpu = $ENV{NUMBER_OF_PROCESSORS} unless $ncpu;
         ($cpu_type) = $Win32::TieRegistry::Registry->{ $idkey } =~ /^(\S+)/;
     }
 
