--- CPAN-Reporter-1.2015/t/06_prompt_text.t~	2015-08-21 23:53:46.000000000 +0200
+++ CPAN-Reporter-1.2015/t/06_prompt_text.t	2016-02-28 11:00:08.000000000 +0100
@@ -19,7 +19,10 @@
 
 BEGIN {
     BEGIN { if (not $] < 5.006 ) { warnings->unimport('redefine') } }
-    *Config::STORE = sub { $_[0]->{$_[1]} = $_[2] }
+    if ($Config{usecperl}) {
+      plan skip_all => "cperl Config is readonly";
+    }
+    *Config::STORE = sub { $_[0]->{$_[1]} = $_[2] };
 }
 
 # For these tests, hide perl_patchlevel so all prompts are tested
