diff -bu ./lib/Dancer/Plugin.pm~ ./lib/Dancer/Plugin.pm
--- ./lib/Dancer/Plugin.pm~	2015-11-07 22:49:34.000000000 +0100
+++ ./lib/Dancer/Plugin.pm	2017-05-19 13:52:27.000000000 +0200
@@ -90,6 +90,7 @@
     {
         no strict 'refs';
         # tried to use unshift, but it yields an undef warning on $plugin (perl v5.12.1)
+        Internals::SvREADONLY(@{"${plugin}::ISA"}, 0);
         @{"${plugin}::ISA"} = ('Dancer::Plugin', @{"${plugin}::ISA"});
         # this works because Dancer::Plugin already ISA Exporter
         push @{"${plugin}::EXPORT"}, @symbols;
