diff -bu DBIx-Class-0.082841-1/Makefile.PL~ DBIx-Class-0.082841-1/Makefile.PL
--- DBIx-Class-0.082841-1/Makefile.PL~	2018-01-29 08:31:50.000000000 +0100
+++ DBIx-Class-0.082841-1/Makefile.PL	2018-09-10 12:52:19.000000000 +0200
@@ -8,21 +8,6 @@
 # during the modern-perl-era
 BEGIN { push @INC, '.' unless grep { $_ eq '.' } @INC }
 
-
-# For the time being simply generate UNKNOWN cperl test reports
-# I would *LOVE* to be able to support it seamlessly, but with
-# rurban adamant that CPAN ought to be gentrified... pity.
-# https://irclog.perlgeek.de/perl11/2016-06-08#i_12624929 ( 06:57 ~ 09:34 )
-BEGIN {
-  die <<'EOE' if $^V =~ /c/;
-
-Currently DBIx::Class is not attempting to be compatible with CPerl
-For more info: https://irclog.perlgeek.de/perl11/2016-06-08#i_12624929
-
-EOE
-}
-
-
 use inc::Module::Install 1.06;
 BEGIN { makemaker_args( NORECURS => 1 ) } # needs to happen early for old EUMM
 
