diffbk Devel-CheckLib-0.92

[CPAN #67300] space in pwd

diff -u Devel-CheckLib-0.92/lib/Devel/CheckLib.pm~ Devel-CheckLib-0.92/lib/Devel/CheckLib.pm
--- Devel-CheckLib-0.92/lib/Devel/CheckLib.pm~	2010-12-12 16:50:50.000000000 +0100
+++ Devel-CheckLib-0.92/lib/Devel/CheckLib.pm	2011-04-07 13:38:27.694118400 +0200
@@ -255,7 +255,7 @@
         }
         warn "# @sys_cmd\n" if $args{debug};
         my $rv = $args{debug} ? system(@sys_cmd) : _quiet_system(@sys_cmd);
-        push @missing, $header if $rv != 0 || ! -x $exefile; 
+        push @missing, $header if $rv != 0 || ! -x $exefile;
         _cleanup_exe($exefile);
         unlink $cfile;
     } 
@@ -308,7 +308,9 @@
         warn "# @sys_cmd\n" if $args{debug};
         my $rv = $args{debug} ? system(@sys_cmd) : _quiet_system(@sys_cmd);
         push @missing, $lib if $rv != 0 || ! -x $exefile;
-        push @wrongresult, $lib if $rv == 0 && -x $exefile && system(File::Spec->rel2abs($exefile)) != 0; 
+        my $absexefile = File::Spec->rel2abs($exefile);
+        $absexefile = '"'.$absexefile.'"' if $absexefile =~ m/\s/;
+        push @wrongresult, $lib if $rv == 0 && -x $exefile && system($absexefile) != 0;
         _cleanup_exe($exefile);
     } 
     unlink $cfile;
