diff -bu ./lib/DOCSIS/ConfigFile/Decode.pm~ ./lib/DOCSIS/ConfigFile/Decode.pm
--- ./lib/DOCSIS/ConfigFile/Decode.pm~	2011-10-03 14:54:16.000000000 -0500
+++ ./lib/DOCSIS/ConfigFile/Decode.pm	2013-07-25 08:21:47.953927842 -0500
@@ -306,7 +306,7 @@
 =cut
 
 sub ip {
-    return inet_ntoa($_[0]) or confess 'inet_ntoa(...) failed to unpack binary string';
+    return inet_ntoa($_[0]) || confess 'inet_ntoa(...) failed to unpack binary string';
 }
 
 =head2 ether
