diff -bu ./hparser.c~ ./hparser.c
--- ./hparser.c~	2016-01-19 16:18:55.000000000 +0100
+++ ./hparser.c	2017-03-01 23:14:12.000000000 +0100
@@ -332,7 +332,7 @@
 
     /* At this point we have decided to generate an event callback */
 
-    argspec = h->argspec ? SvPV(h->argspec, my_na) : "";
+    argspec = h->argspec ? SvPV(h->argspec, my_na) : (char*)"";
 
     if (SvTYPE(h->cb) == SVt_PVAV) {
 
diff -bu ./util.c~ ./util.c
--- ./util.c~	2013-10-21 07:44:45.000000000 +0200
+++ ./util.c	2017-03-01 23:13:29.000000000 +0100
@@ -101,7 +101,7 @@
 	    if (s < end && (*s == 'x' || *s == 'X')) {
 		s++;
 		while (s < end) {
-		    char *tmp = strchr(PL_hexdigit, *s);
+		    const char *tmp = strchr(PL_hexdigit, *s);
 		    if (!tmp)
 			break;
 		    num = num << 4 | ((tmp - PL_hexdigit) & 15);
