diff -bu IO-Compress-2.072-0/Makefile.PL~ IO-Compress-2.072-0/Makefile.PL
--- IO-Compress-2.072-0/Makefile.PL~	2017-02-04 19:24:32.000000000 +0100
+++ IO-Compress-2.072-0/Makefile.PL	2017-02-18 12:11:51.000000000 +0100
@@ -12,6 +12,10 @@
 UpDowngrade(getPerlFiles('MANIFEST'))
     unless $ENV{PERL_CORE};
 
+if ($] > 5.010) {
+    rename 't/Test', 't/Test~';
+}
+
 WriteMakefile( 
     NAME         => 'IO::Compress',
     VERSION_FROM => 'lib/IO/Compress/Base.pm',
