diff -bu Module-Build-0.4216-rvaEm1/lib/Module/Build/Base.pm~ Module-Build-0.4216-rvaEm1/lib/Module/Build/Base.pm
--- Module-Build-0.4216-rvaEm1/lib/Module/Build/Base.pm~	2016-01-20 10:48:08.000000000 +0100
+++ Module-Build-0.4216-rvaEm1/lib/Module/Build/Base.pm	2016-04-17 11:56:40.000000000 +0200
@@ -1079,6 +1079,7 @@
   open(my $fh, '>', $filename) or die "Can't create $filename: $!";
   print $fh <<EOF;
 package $opts{class};
+use lib ".";
 use $pack;
 \@ISA = qw($pack);
 $opts{code}
