diff -bu Moose-2.2006-14/t/bugs/create_anon_role_pass.t~ Moose-2.2006-14/t/bugs/create_anon_role_pass.t
--- Moose-2.2006-14/t/bugs/create_anon_role_pass.t~	2017-07-12 20:07:14.000000000 +0200
+++ Moose-2.2006-14/t/bugs/create_anon_role_pass.t	2017-08-14 10:45:08.000000000 +0200
@@ -5,6 +5,7 @@
 use Test::Fatal;
 use Moose ();
 
+plan skip_all => "no invalid role exception yet with cperl" if $^V =~ /\.27\.\dc$/;
 use lib 't/lib';
 
 {
diff -bu Moose-2.2006-14/t/cmop/add_method_debugmode.t~ Moose-2.2006-14/t/cmop/add_method_debugmode.t
--- Moose-2.2006-14/t/cmop/add_method_debugmode.t~	2017-07-12 20:07:14.000000000 +0200
+++ Moose-2.2006-14/t/cmop/add_method_debugmode.t	2017-08-14 11:00:24.000000000 +0200
@@ -13,6 +13,7 @@
 # preserves the correct debugging information when it (re)adds methods
 # back into a class or role.
 
+plan skip_all => "broken cperl debugger" if $^V =~ /\.2[4-8]\.\dc$/;
 BEGIN {
      $^P = 831; # Enable debug mode
 }
