diff -bu namespace-clean-0.27-0/t/07-debugger.t~ namespace-clean-0.27-0/t/07-debugger.t
--- namespace-clean-0.27-0/t/07-debugger.t~	2016-04-10 05:21:21.000000000 +0200
+++ namespace-clean-0.27-0/t/07-debugger.t	2019-07-05 08:14:48.000000000 +0200
@@ -9,6 +9,9 @@
   ) {
     plan skip_all => $missing_xs;
   }
+  elsif ($^V =~ /c$/) {
+    plan skip_all => "cperl -d";
+  }
   else {
     plan tests => 4;
   }
diff -bu namespace-clean-0.27-0/t/07-pseudo-debugger.t~ namespace-clean-0.27-0/t/07-pseudo-debugger.t
--- namespace-clean-0.27-0/t/07-pseudo-debugger.t~	2016-04-10 05:21:21.000000000 +0200
+++ namespace-clean-0.27-0/t/07-pseudo-debugger.t	2019-07-05 08:16:28.000000000 +0200
@@ -26,7 +26,7 @@
     }
 }
 
-ok( !Foo->can("foo"), "foo cleaned up" );
-ok( !Foo->can("baz"), "baz cleaned up" );
+&Test::More::ok( !Foo->can("foo"), "foo cleaned up" );
+&Test::More::ok( !Foo->can("baz"), "baz cleaned up" );
 
 Foo->bar();
