diff -bu Net-SSLeay-1.72-5MBPMy/t/local/32_x509_get_cert_info.t~ Net-SSLeay-1.72-5MBPMy/t/local/32_x509_get_cert_info.t
--- Net-SSLeay-1.72-5MBPMy/t/local/32_x509_get_cert_info.t~	2013-06-08 00:24:57.000000000 +0200
+++ Net-SSLeay-1.72-5MBPMy/t/local/32_x509_get_cert_info.t	2016-01-12 17:27:57.000000000 +0100
@@ -175,7 +175,7 @@
   }
     
   SKIP: {
-    skip('crl_distribution_points requires 0.9.7+', scalar(@{$dump->{$f}->{cdp}})+1) unless Net::SSLeay::SSLeay >= 0x0090700f;
+    skip('crl_distribution_points requires 0.9.7+', int(@{$dump->{$f}->{cdp}})+1) unless Net::SSLeay::SSLeay >= 0x0090700f;
     my @cdp = Net::SSLeay::P_X509_get_crl_distribution_points($x509);
     is(scalar(@cdp), scalar(@{$dump->{$f}->{cdp}}), "cdp size\t$f");
     for my $i (0..$#cdp) {
