difforig Package-Stash-XS-0.28-b0FyT6

diff -u  Package-Stash-XS-0.28-b0FyT6/t/00-compile.t.orig
--- Package-Stash-XS-0.28-b0FyT6/t/00-compile.t.orig	2016-06-09 11:41:49.000000000 +0200
+++ Package-Stash-XS-0.28-b0FyT6/t/00-compile.t	2016-06-09 11:40:49.000000000 +0200
@@ -58,7 +58,7 @@
     # fake home for cpan-testers
     # no fake requested ## local $ENV{HOME} = tempdir( CLEANUP => 1 );
 
-    like( qx{ $^X -Mlib -e "require $_; print '$_ ok'" }, qr/^\s*$_ ok/s, "$_ loaded ok" )
+    like( qx{ $^X -Mblib -e "require $_; print '$_ ok'" }, qr/^\s*$_ ok/s, "$_ loaded ok" )
         for sort @modules;
 
     SKIP: {
diff -u  Package-Stash-XS-0.28-b0FyT6/t/addsub.t.orig
--- Package-Stash-XS-0.28-b0FyT6/t/addsub.t.orig	2013-07-16 18:17:11.000000000 +0200
+++ Package-Stash-XS-0.28-b0FyT6/t/addsub.t	2016-06-09 11:45:46.000000000 +0200
@@ -40,7 +40,7 @@
     last_line_num  => 199
 );
 
-is $DB::sub{'Foo::dunk'}, sprintf "%s:%d-%d", "FileName", 100, 199,
+is $DB::sub{'Foo::dunk'}, sprintf("%s:%d-%d", "FileName", 100, 199),
     '... got the right %DB::sub value for dunk with specified args';
 
 done_testing;
