diff -bu ./t/toomany.t~ ./t/toomany.t
--- ./t/toomany.t~	2017-05-15 11:41:22.000000000 +0200
+++ ./t/toomany.t	2017-05-15 11:35:56.000000000 +0200
@@ -29,7 +29,7 @@
     my @all_names =
         ($name, map "$_\::$name", qw<Ref::Util Ref::Util::PP Ref::Util::XS>);
     my $rx = join '|', (
-        (map "Too many arguments for $_\\b", @all_names),
+        (map "Too many arguments for( subroutine entry)? $_\\b", @all_names),
         (map "Usage: $_\\(ref\\)", @all_names),
     );
     like($exn, qr/^(?:$rx)/, $desc);
