difforig Socket-1.99-Osq8ZG

diff -u  Socket-1.99-Osq8ZG/Socket.xs.orig
--- Socket-1.99-Osq8ZG/Socket.xs.orig	2012-02-16 18:14:33.000000000 -0600
+++ Socket-1.99-Osq8ZG/Socket.xs	2012-03-07 15:43:47.600312241 -0600
@@ -713,9 +713,15 @@
 	if (sockaddrlen != sizeof(addr))
 		croak("Bad arg length for %s, length is %"UVuf", should be %"UVuf,
 		      "Socket::unpack_sockaddr_un", (UV)sockaddrlen, (UV)sizeof(addr));
-#   endif
-
 	Copy(sun_ad, &addr, sizeof(addr), char);
+#   else
+	if (sockaddrlen < sizeof(addr)) {
+	  Copy(sun_ad, &addr, sockaddrlen, char);
+	  Zero(&addr+sockaddrlen, sizeof(addr)-sockaddrlen, char);
+	} else {
+	  Copy(sun_ad, &addr, sizeof(addr), char);
+	}
+#   endif
 
 	if (addr.sun_family != AF_UNIX)
 		croak("Bad address family for %s, got %d, should be %d",
