diff -bu Sub-Install-0.928-1/lib/Sub/Install.pm~ Sub-Install-0.928-1/lib/Sub/Install.pm
--- Sub-Install-0.928-1/lib/Sub/Install.pm~	2014-06-27 00:57:41.000000000 +0200
+++ Sub-Install-0.928-1/lib/Sub/Install.pm	2018-09-08 14:15:20.336382414 +0200
@@ -114,7 +114,8 @@
     Carp::croak "couldn't determine name under which to install subroutine"
       unless $arg->{as};
 
-    $installer->(@$arg{qw(into as code) });
+    my @args = @$arg{qw(into as code) };
+    $installer->(@args);
   }
 }
 
diff -bu Sub-Install-0.928-1/t/install.t~ Sub-Install-0.928-1/t/install.t
--- Sub-Install-0.928-1/t/install.t~	2014-06-27 00:57:41.000000000 +0200
+++ Sub-Install-0.928-1/t/install.t	2018-09-08 14:14:45.764694137 +0200
@@ -33,6 +33,12 @@
 
     $stderr =~ s!\.$!!gm;
     $stderr =~ s!\\!/!g;
+    if ($^V =~ /c$/) {
+      $expected_warning = <<'END_WARNING';
+Subroutine main::tmp_ok redefined at t/install.t line 31
+Prototype mismatch: sub main::tmp_ok: none vs ($$;$) at t/install.t line 31
+END_WARNING
+    }
     is(
       $stderr,
       $expected_warning,
diff -bu Sub-Install-0.928-1/t/reinstall.t~ Sub-Install-0.928-1/t/reinstall.t
--- Sub-Install-0.928-1/t/reinstall.t~	2014-06-27 00:57:41.000000000 +0200
+++ Sub-Install-0.928-1/t/reinstall.t	2018-09-08 14:23:38.007938489 +0200
@@ -25,8 +25,13 @@
     Sub::Install::reinstall_sub({ code => \&ok, as => 'tmp_ok' });
 
     my $expected_warning = <<'END_WARNING';
-Prototype mismatch: sub main::tmp_ok ($;$) vs ($$;$) at t/reinstall.t line 32
+Prototype mismatch: sub main::tmp_ok ($;$) vs ($$;$) at t/reinstall.t line 37
 END_WARNING
+    if ($^V =~ /c$/) {
+      $expected_warning = <<'END_WARNING';
+Prototype mismatch: sub main::tmp_ok: none vs ($$;$) at t/reinstall.t line 37
+END_WARNING
+    }
 
     my $stderr = Test::Output::stderr_from(
       sub { Sub::Install::reinstall_sub({ code => \&is, as => 'tmp_ok' }) }
