diff -bu ./lib/Test/utf8.pm~ ./lib/Test/utf8.pm
--- ./lib/Test/utf8.pm~	2014-01-13 14:14:36.000000000 +0100
+++ ./lib/Test/utf8.pm	2018-01-09 11:48:06.000000000 +0100
@@ -314,8 +314,7 @@
 The opposite of C<is_flagged_utf8>, passes if and only if the string
 isn't flagged as utf8 by perl's internals.
 
-Note: you can refer to this function as C<isn't_flagged_utf8> if you
-really want to.
+Note: C<isn't_flagged_utf8> is invalid since v5.26.0c
 
 =cut
 
diff -bu ./t/01basic.t~ ./t/01basic.t
--- ./t/01basic.t~	2004-09-10 11:29:11.000000000 +0200
+++ ./t/01basic.t	2018-01-09 11:48:34.000000000 +0100
@@ -15,4 +15,4 @@
 ok(defined(&is_within_latin1),       "is_within_latin1 imported");
 ok(defined(&is_flagged_utf8),        "is_flagged_utf8 imported");
 ok(defined(&isnt_flagged_utf8),      "isnt_flagged_utf8 imported");
-ok(defined(&isn't_flagged_utf8),     "isn't_flagged_utf8 imported");
+ok(defined(&isn::t_flagged_utf8),     "isn't_flagged_utf8 imported");
diff -bu ./t/06flag.t~ ./t/06flag.t
--- ./t/06flag.t~	2004-09-10 11:29:12.000000000 +0200
+++ ./t/06flag.t	2018-01-09 11:48:59.000000000 +0100
@@ -38,14 +38,14 @@
 ######################
 
 test_out("ok 1 - not flagged as utf8");
-isn't_flagged_utf8("fred");
+isn::t_flagged_utf8("fred");
 test_test("isn't flagged pass");
 
 test_out("ok 1 - foo");
-isn't_flagged_utf8("fred","foo");
+isn::t_flagged_utf8("fred","foo");
 test_test("isn't flagged pass with name");
 
 test_out("not ok 1 - not flagged as utf8");
 test_fail(+1);
-isn't_flagged_utf8("\x{400}");
+isn::t_flagged_utf8("\x{400}");
 test_test("isn't flagged fail");
