diffbk Test-NoTabs-1.0-IVZpfk

[cpan #67376] space in pwd

diff -u Test-NoTabs-1.0-IVZpfk/t/12-fail.t~ Test-NoTabs-1.0-IVZpfk/t/12-fail.t
--- Test-NoTabs-1.0-IVZpfk/t/12-fail.t~	2009-07-28 10:37:32.000000000 +0200
+++ Test-NoTabs-1.0-IVZpfk/t/12-fail.t	2011-04-11 12:54:06.244166200 +0200
@@ -5,8 +5,8 @@
 use File::Temp qw( tempdir tempfile );
 
 my $perl  = $^X || 'perl';
-my $inc = join(' -I ', @INC) || '';
-$inc = "-I $inc" if $inc;
+$perl = $perl =~ m/\s/ ? qq{"$perl"} : $perl;
+my $inc = "-I blib/arch -I blib/lib";
 
 {
     my $dir = make_bad_file_1();
