diff -bu WebService-Rackspace-CloudFiles-1.09-JSvaai/lib/WebService/Rackspace/CloudFiles.pm~ WebService-Rackspace-CloudFiles-1.09-JSvaai/lib/WebService/Rackspace/CloudFiles.pm
--- WebService-Rackspace-CloudFiles-1.09-JSvaai/lib/WebService/Rackspace/CloudFiles.pm~	2013-07-17 10:20:57.000000000 -0500
+++ WebService-Rackspace-CloudFiles-1.09-JSvaai/lib/WebService/Rackspace/CloudFiles.pm	2013-07-23 12:47:29.039280343 -0500
@@ -42,7 +42,7 @@
     default  => sub {
         my $self = shift;
 
-        return $self->locations->{$self->location} or
+        return $self->locations->{$self->location} ||
             confess "location $self->{location} unknown: valid locations are " .
                 join ', ', $self->location_names ;
     },
