diff -bu YAML-Tiny-1.69-jGSzJX/t/lib/SubtestCompat.pm~ YAML-Tiny-1.69-jGSzJX/t/lib/SubtestCompat.pm
--- YAML-Tiny-1.69-jGSzJX/t/lib/SubtestCompat.pm~	2015-07-26 02:51:17.000000000 +0200
+++ YAML-Tiny-1.69-jGSzJX/t/lib/SubtestCompat.pm	2016-06-10 01:05:58.000000000 +0200
@@ -49,7 +49,7 @@
         if ( my $err = $@ ) {
             if ( ref($err) eq 'Subtest::SKIP' ) {
                 SKIP: {
-                    skip $err->{reason}, 1;
+                    skip "$err->{reason}", 1;
                 }
             }
             else {
